lib/inets/src/http_server/httpd_request_handler.erl

```diff
@@ -327,16 +327,16 @@ do_terminate(#state{mod = ModData} = State) ->
     httpd_socket:close(ModData#mod.socket_type, ModData#mod.socket).
 
 format_status(normal, [_, State]) ->
-    [{data, [{"StateData", State}]}];  
+    [{data, [{"StateData", State}]}];
 format_status(terminate, [_, State]) ->
     Mod = (State#state.mod),
     case Mod#mod.socket_type of
-	ip_comm ->
-	    [{data, [{"StateData", State}]}];  
-	{essl, _} ->
+	{ssl, _} ->
 	    %% Do not print ssl options in superviosr reports
-	    [{data, [{"StateData", 
-		      State#state{mod = Mod#mod{socket_type = 'TLS'}}}]}]
+	    [{data, [{"StateData",
+		      State#state{mod = Mod#mod{socket_type = 'TLS'}}}]}];
+        _  ->
```

Comment:
Is it better if we are explicit here to know that the only other expected matching value is `ip_comm`?
It could help us to know that this is the only other case that should happen

Reply:
It is a trade off it could probably be ip_comm or {ip_comm, _Opts} but we do not care we only care if it is ssl or not. I think I will leave this for now, could be part of future refactors the we probably will do soon or later. Probably later ;)